cmd/buildah/pull.go

```diff
@@ -87,6 +88,8 @@ func pullCmd(c *cli.Context) error {
 		return errors.Wrapf(err, "error building system context")
 	}
 
+	utils.CheckConfigFiles()
```

Comment:
I was thinking this call would live in cmd/buildah/main.go up near the top of the main() function.  I just checked and had forgotten that we've a check there for Storage Options already, so probably makes sense to add this type of check there too rather than in each command.

Reply:
Agree, it should be in Main.

Reply:
cmd/buildah/main.go I can only check using the default path?
I'm thinking if the path can override by the user, the method for the command will check it and return errors so we only need to check the default path of the config file required for all commands and maybe that's an error.
but currently, I'm not sure which is must be required